docs/content/preview/explore/cluster-management/point-in-time-recovery-ycql.md

```diff
@@ -104,7 +104,7 @@ Create and populate a table, get a timestamp to which you'll restore, and then w
 1. Verify that a snapshot has happened:
 
     ```sh
-    ./bin/yb-admin list_snapshot_schedules
+    ./bin/yb-admin -master_addresses ip1:7100,ip2:7100,ip3:7100 \ list_snapshot_schedules
```

Comment:
```suggestion
    ./bin/yb-admin -master_addresses <ip1:7100,ip2:7100,ip3:7100> \ list_snapshot_schedules
```

Reply:
In some places it's with `<ip1:7100,ip2:7100,ip3:7100>`, in some places it's `ip1:7100,ip2:7100,ip3:7100`. I'd like it to be consistent, either version works but I'd prefer with `<>`